Release v1.7.5

jakirkham opened the pull request:
Pulled the commit from PR ( https://github.com/conda-forge/sphinx-feedstock/pull/35 ).

Commit:
updated v1.7.5

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

jakirkham committed:
Update dependencies

Add version constraints that were missing previously. Also add any
missing dependencies.

chohner commented:
Thanks!
